plugins/org.eclipse.fordiac.ide.model.eval/src/org/eclipse/fordiac/ide/model/eval/function/StandardFunctions.java

```diff
@@ -485,12 +485,12 @@ static <T extends AnyWCharsValue> AnyIntValue FIND(@Comment("The WString to be s
 	/* Array functions */
 
 	@Comment("Returns the lower bound of the dimension of the given array")
-	static <T extends ArrayValue, U extends AnyIntValue> AnyIntValue LOWER_BOUND(final T array, final U dimension) {
+	static <U extends AnyIntValue> AnyIntValue LOWER_BOUND(final ArrayValue array, final U dimension) {
 		return DIntValue.toDIntValue(array.getType().getSubranges().get(dimension.intValue() - 1).getLowerLimit());
 	}
 
 	@Comment("Returns the upper bound of the dimension of the given array")
-	static <T extends ArrayValue, U extends AnyIntValue> AnyIntValue UPPER_BOUND(final T array, final U dimension) {
+	static <U extends AnyIntValue> AnyIntValue UPPER_BOUND(final ArrayValue array, final U dimension) {
```

Comment:
The exact signature of this method, including the generic parameters, is important for deriving ST declarations.